xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -455,11 +455,15 @@ The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original wri
 
 ## Remarks
 
+The maximum allowed size of the binary data to write as Base64 is 125_000_000 bytes (or ~125MB). Exceeding this limit will result in an <xref:System.ArgumentException> being thrown.
```

Comment:
```suggestion
The maximum allowed size of the binary data to write as Base64 is 125,000,000 bytes (or approximately 125 MB). Exceeding this limit results in an <xref:System.ArgumentException> being thrown.
```

Related guidance: 
- https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms
- https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms